src/mds/Server.cc

```diff
@@ -2063,7 +2063,9 @@ void Server::perf_gather_op_latency(const cref_t<MClientRequest> &req, utime_t l
   case CEPH_MDS_OP_RENAMESNAP:
     code = l_mdss_req_renamesnap_latency;
     break;
-  default: ceph_abort();
+  default:
+    dout(1) << ": unknown client op" << dendl;
+    return;
```

Comment:
@vshankar would it help with debugging if the invalid op value is dumped in the log as well ?

Reply:
That's already done in handle_client_request, so, I omitted that bit here...